processing/src/test/java/org/apache/druid/segment/filter/EqualityFilterTests.java

```diff
@@ -288,6 +310,28 @@ public void testSingleValueStringColumnWithNulls()
             NotDimFilter.of(new EqualityFilter("s0", ColumnType.STRING, "noexist", null)),
             ImmutableList.of("0", "1", "2", "3", "4", "5")
         );
+
+        // in default value mode, is true/is false are basically pointless since they have the same behavior as = and <>
+        // "(s0 = 'a') is not true" equivalent to "s0 <> 'a'"
+        assertFilterMatches(
```

Comment:
why are new assert statements being added?

Reply:
looks like there was a error when rebasing on the updated fork of the master